README.md

```diff
@@ -1,10 +1,6 @@
 # Getting Started with the Vue Scheduler Component 
 
-A quick start project that shows how to add Syncfusion's Vue Scheduler component to the Vue project. This project also includes a code snippet to add appointments to the Scheduler either as local JSON data or remote service URLs, and also to change the default date and view.
-
-Example: https://ej2.syncfusion.com/vue/demos/#/material/schedule/default.html 
-
-Documentation: https://ej2.syncfusion.com/vue/documentation/schedule/getting-started-vue-3/ 
+The Vue [Scheduler](https://www.syncfusion.com/vue-components/vue-scheduler?utm_source=github&utm_medium=listing&utm_campaign=vue-scheduler-github-samples), or event calendar, is a fully featured event calendar component that helps users manage their time efficiently. A quick start project that shows how to add Syncfusion's Vue Scheduler component to the Vue project. This project also includes a code snippet to add appointments to the Scheduler either as local JSON data or remote service URLs, and also to change the default date and view.
 
```

Comment:
Check on this
![image](https://github.com/SyncfusionExamples/getting-started-with-the-vue-schedular/assets/95272441/1821ad08-59c5-4cb2-914e-2cd4dca26500)

Reply:
Corrected